src/main.rs

```diff
@@ -45,6 +46,11 @@ fn main() -> anyhow::Result<()> {
     path.push(format!("{APP_NAME}/{APP_NAME}.db"));
     path
   };
+
+  if fs::metadata(&db_path).is_err() {
```

Comment:
```suggestion
  if !db_path.exists() {
```

Reply:
Thanks! Didn't realize the function was available